src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothGattImpl.cs

```diff
@@ -637,8 +637,13 @@ protected override bool ReleaseHandle()
         {
             if (_hasOwnership == true)
             {
-                Interop.Bluetooth.BtGattServerDeinitialize();
-                Interop.Bluetooth.BtGattServerDestroy(handle);
+                int err;
+
+                err = Interop.Bluetooth.BtGattServerDestroy(handle);
+                GattUtil.ThrowForError(err, "Failed to destroy the server instance");
```

Comment:
Do not throw excaption on ReleaseHandle method
just returing false to inform error condition

https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.safehandle.releasehandle?view=netframework-4.7.2

> the code in your implementation of ReleaseHandle must never fail.

Reply:
Done